### ISSUE TITLE
deprecated constants

### ISSUE THREAD

**Issue by reshadf** [magynhard/class_interface#1]
Hi I am getting these deprecated constants. Are they going to be updated anytime soon?

```/.rvm/gems/ruby-2.6.2/gems/class_interface-0.1.0/lib/class_interface.rb:81: warning: constant ::Bignum is deprecated
/.rvm/gems/ruby-2.6.2/gems/class_interface-0.1.0/lib/class_interface.rb:82: warning: constant ::Fixnum is deprecated
/.rvm/gems/ruby-2.6.2/gems/class_interface-0.1.0/lib/class_interface.rb:81: warning: constant ::Bignum is deprecated
/.rvm/gems/ruby-2.6.2/gems/class_interface-0.1.0/lib/class_interface.rb:82: warning: constant ::Fixnum is deprecated```

**Comment by magynhard** [comment]
Hi @reshadf,

thank you for your issue.

This gem already included a explicit check and support for Ruby > 2.4 and Ruby >= 2.4.
But for sure deprecation warnings are not a nice thing ...

Just released a new version 0.1.1 which should make you (and others) happy! 👍 

Can you please close this issue if the new release makes you happy?

**Comment by reshadf** [comment]
Thanks, @magynhard. it worked! closing this issue.
